.github/actions/spelling/reject.txt

```diff
@@ -1,4 +1,5 @@
 ^attache$
+^bellow$
```

Comment:
This is a typo I've run into in a couple of places (including here!).

`bellow` is a thing used to stoke flames 🔥...